AuroraEditor/Base/MenuBarItem/StatusItem.swift

```diff
@@ -10,6 +10,13 @@ import Foundation
 import SwiftUI
 
 extension AppDelegate {
+    private var appVersion: String {
+        Bundle.versionString ?? "No Version"
+    }
+
+    private var appBuild: String {
+        Bundle.buildString ?? "No Build"
```

Comment:
We use return statements everywhere.

```suggestion
        return Bundle.buildString ?? "No Build"
```